docs/source/install.md

```diff
@@ -173,11 +173,9 @@ FlowAuth is designed to be deployed as a single Docker container working in coop
 
 FlowAuth supports any database supported by [SQLAlchemy](https://sqlalche.me), and to connect you will only need to supply a correct URI for the database either using the `DB_URI` environment variable, or by setting the `DB_URI` secret. If `DB_URI` is not set, a temporary sqlite database will be created.
 
-On first use, you will need to create the necessary tables and an administrator account. 
+FlowAuth will attempt to create all necessary tables when first accessed, but will not overwrite any existing tables. To wipe any existing data, you can either set the `INIT_DB` environment variable to `true`, or run `flask init-db` from inside the container (`docker exec <container-id> flask init-db`).
```

Comment:
Should this be `RESET_FLOWAUTH_DB` instead of `INIT_DB`? (Haven't tried it, just from looking at the changes in `flowauth/config.py`.)